hknweb/events/models.py

```diff
@@ -36,9 +36,9 @@ def __str__(self):
        return self.name
 
 class Rsvp(models.Model): # TODO: null should be false in some cases
-    user  = models.ForeignKey(User, models.CASCADE, null=True, verbose_name="rsvp'd by")
-    event = models.ForeignKey(Event, models.CASCADE, null=True)
-    confirmed       = models.BooleanField(null=True)
+    user  = models.ForeignKey(User, models.CASCADE, null=False, verbose_name="rsvp'd by")
+    event = models.ForeignKey(Event, models.CASCADE, null=False)
+    confirmed       = models.BooleanField(null=False, default=False)
```

Comment:
null=False is actually the default value, fyi.

Reply:
I know but we can be explicit

Reply:
Explicit is (usually) good, but here we're not even consistent with it; we omit it from literally every other field and it's the default.

I feel like consistency trumps explicitness here; on reading this code, usually the (naive) assumption is that the default is the opposite of the explicit value given.